handlers/salesforce-disaster-recovery/src/handlers/saveFailedRowsToS3.ts

```diff
@@ -0,0 +1,49 @@
+import {
+	type AccountRowResult,
+	convertArrayToCsv,
+	getFileFromS3,
+	uploadFileToS3,
+} from '../services';
+
+export const handler = async (event: {
+	resultFiles: Array<{ Key: string; Size: number }>;
+	filePath: string;
+}): Promise<void> => {
+	const { resultFiles, filePath } = event;
+
+	const bucketName = process.env.S3_BUCKET;
+
+	if (!bucketName) {
+		throw new Error('Environment variables not set');
+	}
```

Comment:
```suggestion
const bucketName = checkDefined(
		process.env.S3_BUCKET,
		'Environment variables not set',
	);
```
Up to you, but I find this a bit more readable